src/SFA.DAS.AssessorService.Application/Interfaces/IStandardService.cs

```diff
@@ -7,12 +7,12 @@ namespace SFA.DAS.AssessorService.Application.Interfaces
 {
     public interface IStandardService
     {
-        Task<IEnumerable<StandardSummary>> GetAllStandardsV2();
-        Task<IEnumerable<Standard>> GetAllStandards();
-        Task<IEnumerable<StandardCollation>> GatherAllStandardDetails();
-        Task<Standard> GetStandard(int standardId);
-        Task<Standard> GetStandard(string referenceNumber);
-        Task<IEnumerable<StandardSummary>> GetAllStandardSummaries();
+        Task<IEnumerable<StandardCollation>> GetAllStandardsV2();
```

Comment:
As mentioned earlier do we really want to call this V2?

Reply:
☝️